### DISCUSSION TITLE
error when pulling in blog posts

### DISCUSSION THREAD

**Discussion by juno-junho** [gautamkrishnar/blog-post-workflow#248] [Q&A]
```name: Latest blog post workflow
on:
  schedule: # Run workflow automatically
    - cron: '0 * * * *' # Runs every hour, on the hour
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the Github Actions Workflow page directly
permissions:
  contents: write # To write the generated contents to the readme

jobs:
  update-readme-with-blog:
    name: Update this repo's README with latest blog posts
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4
      - name: Pull in tistory posts
        uses: gautamkrishnar/blog-post-workflow@v1
        with:
          feed_list: "https://juno-juno.tistory.com/rss"
```

This is my workflow.yml file. 
There's an error when pulling in blog posts. 
I am using 'Tistory', a widely used blog site in South Korea. 
I've entered the correct RSS page in the feed list, but it doesn't work. It used to work, but it hasn't been functioning for the past two weeks. 
Can you help me figure out what's wrong?

<img width="1253" alt="image" src="https://github.com/gautamkrishnar/blog-post-workflow/assets/72647031/4ef4dc4c-6db6-4279-a0b1-4b5813a29a56">

**Comment by gautamkrishnar** [comment]
@juno-junho  https://github.com/gautamkrishnar/blog-post-workflow?tab=readme-ov-file#faqs-and-common-issues

**Reply by juno-junho** [reply]
thanks for notice. ☺️
